iroh/src/run.rs

```diff
@@ -62,6 +54,20 @@ enum Commands {
         /// filesystem path to write to. Optional and defaults to $CID
         output: Option<PathBuf>,
     },
+    #[clap(about = "Start iroh services locally")]
+    #[clap(after_help = doc::START_LONG_DESCRIPTION )]
+    Start {},
+    // status checks the health of the different processes
```

Comment:
I see it was already two `//` before it got moved, but if we want this to be in the CLI help we'll need `///`.